tox.ini

```diff
@@ -216,7 +216,7 @@ setenv =
     PYVISTA_OFF_SCREEN = true
     PYTEST_ADDOPTS = --color=yes -v {env:PARALLEL:}
 commands =
-	modules: pytest --doctest-modules {env_site_packages_dir}{/}pyvista {posargs}
+	modules: pytest --doctest-modules --doctest-continue-on-failure {env_site_packages_dir}{/}pyvista {posargs}
```

Comment:
Found convenient to use `--doctest-continue-on-failure` since it enables to see all docstring issues and not only the first one.